numpy/lib/function_base.py

```diff
@@ -1856,13 +1856,26 @@ def diff(a, n=1, axis=-1):
         The number of times values are differenced.
     axis : int, optional
         The axis along which the difference is taken, default is the last axis.
+    prepend : array_like
+        Values to insert along axis before the differenced results.  'first' is
+        an alias for a.take([0], axis), which inserts the first value from a 
```

Comment:
I doubt that would be used much at all.  'first' is much more common.  Also its a really simple recipe, just as you show.  I don't think the alias 'last' should be added